File: lib_httpmock_no_op/src/main/java/com/estyle/httpmock/HttpMock.java
package com.estyle.httpmock;

import android.content.Context;

import okhttp3.OkHttpClient;

public class HttpMock {

    public static OkHttpClient.Builder addHttpMockInterceptor(
            OkHttpClient.Builder builder,
            Context context,
            boolean enable,
            Class<HttpMockGenerator> generatorClass
    ) {
        return builder;
    }
}

File: lib_httpmock_no_op_ktx/src/main/java/com/estyle/httpmock/HttpMock.kt
package com.estyle.httpmock

import android.content.Context
import okhttp3.OkHttpClient

fun OkHttpClient.Builder.addHttpMockInterceptor(
    context: Context,
    enable: Boolean,
    delayMillis: Long,
    generatorClass: Class<HttpMockGenerator>
) = this